docs/millennium-os/manual/chapters/40_gcodes.md

```diff
@@ -12,7 +12,7 @@
 G27 [Zn]
 ```
 
-Used to park the spindle in a safe location, by default it will move the spindle to the top of the Z axis, trigger an `M5.9` (stop spindle and wait for spin-down), and then move the table to the centre of the X axis and at the front (towards the operator) of the Y axis.
+Used to park the spindle in a safe location, by default it will move the spindle to the top of the Z-Axis, trigger an `M5.9` (stop the spindle and wait for spin-down), and then move the table to the centre of the X-Axis and at the front (towards the operator) of the Y-Axis.
```

Comment:
```suggestion
Used to park the spindle in a safe location, by default it will move the spindle to the top of the Z-Axis, trigger an `M5.9` (stop the spindle and wait for spin-down), and then move the table to the centre of the X-Axis and at the front of the Y-Axis (towards the operator).
```